src/main/java/org/esbtools/gateway/resync/exception/IncompleteRequestException.java

```diff
@@ -0,0 +1,15 @@
+package org.esbtools.gateway.resync.exception;
+
+import org.esbtools.gateway.resync.ResyncRequest;
+import org.esbtools.gateway.resync.service.ResyncErrorMessages;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value= HttpStatus.BAD_REQUEST, reason=ResyncErrorMessages.INCOMPLETE_REQUEST)
```

Comment:
Test have been converted to using Spring MVC unit test framework, and the resync method is now 3 lines of code (Yeah!)